### PR TITLE
fix update_version.sh on macos

### PR DESCRIPTION
**Pull request by daiyam**
This PR adds `VSCODE_ARCH` to the archive and to `VERSION_PATH ` but not sure if it's correct...

### CODE REVIEW

File: update_version.sh [hunk 1]
```diff
@@ -99,8 +99,8 @@ cd ..
 
 if [[ "$OS_NAME" == "osx" ]]; then
   # zip, sha1, and sha256 files are all at top level dir
-  ASSET_NAME=VSCodium-darwin-${LATEST_MS_TAG}.zip
-  VERSION_PATH="darwin"
+  ASSET_NAME=MrCode-darwin-${VSCODE_ARCH}-${LATEST_MS_TAG}.zip
```
Comment (stripedpajamas) [inline update_version.sh:102]: Should be VSCodium not MrCode

Comment (daiyam) [reply update_version.sh]: Oops... fixed

**Comment by daiyam** [comment]
@stripedpajamas I've updated `update-api` to add a default arch (x64) for macos. So now, this PR is correct.

**Review by stripedpajamas — APPROVED** [review]
Another good catch :D thank you

**Pull request merged**